wrapper/pyproject.toml

```diff
@@ -63,10 +63,6 @@ universal = true
 # Developer tool configurations
 #
 
-# Disable black
-[tool.black]
-exclude = ".*"
```

Comment:
Please leave this block, and change the block in the main package to also disable black. This should prevent accidental reformats by eager editors.